Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -129,6 +154,36 @@ public ICommand ItemClickCommand
         }
 
         /// <summary>
+        /// Gets or sets the selected multiple objects in the collection.
+        /// </summary>
+        /// <value>The object that is used to store selected multiple items. Default is an empty IList collection of object.</value>
+        public IList<object> SelectedItems
+        {
+            get { return (IList<object>)GetValue(SelectedItemsProperty); }
+            set { SetValue(SelectedItemsProperty, value); }
```

Comment:
What does happen if I set it manually? It does not seem to affect the internal listview